hooks/react-client/useKeyboardCommands.ts

```diff
@@ -27,9 +27,11 @@ const useKeyboardCommands = (fn: KeyboardCommandCallback) => {
           fn('up');
           break;
       }
-    });
+    }
 
-    return () => document.removeEventListener('keydown', () => {});
+    document.addEventListener('keydown', handleKeyDown);
+
+    return () => document.removeEventListener('keydown', handleKeyDown);
```

Comment:
Nice cleanup! That removeEventListener was doing nothing. Not sure how this got passed on code reviews 🤦 

But instead of using regular functions. Could you use an arrow function?